.gitignore

```diff
@@ -62,3 +62,4 @@ cmake-build-*/
 
 # Vim
 *.swp
+tags
```

Comment:
what are tags?

Reply:
For the record, possibly related to [ctags](https://en.wikipedia.org/wiki/Ctags) (or at least a similar tool), which is an old mechanism for code browsing (i.e. searching uses of functions, jumping from a function use to its identifier... i.e. the stuff taken for granted in most modern IDEs, except it's been around longer and can be integrated with e.g. vim).

Reply:
That's correct. I use ctags for my Vim setup.